pkg/reference/handler.go

```diff
@@ -6,25 +6,27 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-func GetRequests(a client.Object, source interface{}) []reconcile.Request {
+//
+func GetRequests(a client.Object, ownerNamespace string, source interface{}) []reconcile.Request {
```

Comment:
NIT: s/ownerNamespace/migrationNamespace? 

technically this is not an owner namespace this is always going to be a migrationNamesace right?